asllib/tests/regressions.t/stdlib.asl

```diff
@@ -150,8 +150,7 @@ begin
     let expected_res = SqrtRounded(x, 1000);
     for p = 1 to 10 do
       let res = SqrtRounded(x, p);
-      // +2 because of the rounding to odd?
-      assert Abs(res - expected_res) / expected_res <= 2.0 ^ (-p+2);
+      assert Abs(res - expected_res) / expected_res <= 2.0 ^ (1-p);
```

Comment:
I think this bound could be 2.0^(-p) and be strict. E.g., with p=1 the biggest relative error should be when the expected result is very close to 1 and the computed result is 1.5 so relative error is nearly but less than 0.5.